conn.go

```diff
@@ -76,15 +76,11 @@ func (c *Conn) doTransform(tree pg_query.ParsetreeList) (string, int, error) {
 }
 
 func normalize(q string) string {
-	lines := strings.Split(q, "\n")
-	for i, line := range lines {
-		lines[i] = strings.TrimSpace(line)
-	}
-	for len(lines) > 0 && len(lines[0]) == 0 {
-		lines = lines[1:]
-	}
-	for len(lines) > 0 && len(lines[len(lines)-1]) == 0 {
-		lines = lines[:len(lines)-1]
+	lines := []string{}
```

Comment:
```suggestion
	var lines []string{}
```

The difference here is that `lines := []string{}` allocates the (nonzero) space needed for a zero-length slice, and makes `lines` point to that. Setting `lines` to `nil` instead, which is what my suggestion does, avoids allocating anything until/unless needed.